src/vm/nativefieldflags.h

```diff
@@ -0,0 +1,40 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// Describes specific categories/subcategories of native fields.
+enum NativeFieldFlags : short
+{
+    // The native representation of the field is a floating point field.
+    NATIVE_FIELD_SUBCATEGORY_FLOAT = 1 << 8,
+    // The field has a nested MethodTable* (i.e. a field of a struct, class, or array)
+    NATIVE_FIELD_SUBCATEGORY_NESTED = 1 << 9,
+    // The field has a type that is only relevant in COM and should never be marshalled on non-COM ABIs.
```

Comment:
Love me some double negatives, but communicate this in a way that matches the enum name - "The field has a type that is only relevant in COM and requires adherence to the COM ABI."